tempo_project/dimensions/supplemental/transportation_end_uses.csv

```diff
@@ -0,0 +1,3 @@
+id,name,fuel_id,unit
+electricity_ev_l1l2,EV L1/L2,electricity,MWh
+electricity_ev_dcfc,EV DCFC,electricity,MWh
```

Comment:
Delete this file, subsectors_by_sector, electric_end_uses, other transportation_* files.